src/probabilities_estimators/dispersion/dispersion.jl

```diff
@@ -0,0 +1,172 @@
+
+julia> c, m = 3, 5;
+
+julia> est = Dispersion(s = GaussianSymbolization(c = c), m = m);
+
+julia> probs = probabilities(x, est)
+9-element Probabilities{Float64}:
+ 0.09375000000000001
+ 0.10416666666666669
+ 0.18750000000000003
+ 0.09375000000000001
+ 0.10416666666666669
+ 0.10416666666666669
+ 0.10416666666666669
+ 0.10416666666666669
+ 0.10416666666666669
+```
+
+Note that dispersion patterns that are not present are not counted. Therefore, you'll
+always get non-zero probabilities using the `Dispersion` probability estimator.
+
+To compute (normalized) dispersion entropy of order `q` to a given `base` on the
+univariate input time series `x`, do:
+
+```jldoctest dispersion_example
+julia> entropy_renyi(x, Dispersion(normalize = true), base = 2, q = 1)
+0.6716889280681666
+```
+
+If `normalize == true`, then when used in combination with [`entropy_renyi`](@ref),
+the dispersion entropy is normalized to `[0, 1]`. Normalization is only
+defined when `q == 1`.
+
+## Data requirements and parameters
+
+The input must have more than one unique element for the Gaussian mapping to be
+well-defined. Li et al. (2018) recommends that `x` has at least 1000 data points.
+
+If `check_unique == true` (default), then it is checked that the input has
+more than one unique value. If `check_unique == false` and the input only has one
+unique element, then a `InexactError` is thrown when trying to compute probabilities.
+
+See also: [`entropy_dispersion`](@ref), [`GaussianSymbolization`](@ref).
+
+
+!!! note
+    ## Why "dispersion patterns"?
+    Each embedding vector is called a "dispersion pattern". Why? Let's consider the case
+    when ``m = 5`` and ``c = 3``, and use some very imprecise terminology for illustration:
+
+    When ``c = 3``, values clustering far below mean are in one group, values clustered
+    around the mean are in one group, and values clustering far above the mean are in a
+    third group. Then the embedding vector ``[2, 2, 2, 2, 2]`` consists of values that are
+    relatively close together (close to the mean), so it represents a set of numbers that
+    are not very spread out (less dispersed). The embedding vector ``[1, 1, 2, 3, 3]``,
+    however, represents numbers that are much more spread out (more dispersed), because the
+    categories representing "outliers" both above and below the mean are represented,
+    not only values close to the mean.
+
+[^Rostaghi2016]: Rostaghi, M., & Azami, H. (2016). Dispersion entropy: A measure for time-series analysis. IEEE Signal Processing Letters, 23(5), 610-614.
+[^Li2018]: Li, G., Guan, Q., & Yang, H. (2018). Noise reduction method of underwater acoustic signals based on CEEMDAN, effort-to-compress complexity, refined composite multiscale dispersion entropy and wavelet threshold denoising. Entropy, 21(1), 11.
+"""
+Base.@kwdef struct Dispersion <: ProbabilitiesEstimator
+    s = GaussianSymbolization(c = 5)
+    m = 2
+    τ = 1
+    check_unique = false
+    normalize = true
+end
```

Comment:
Why is it that none of the fields here is type stable...? I don't think that was intended, right? All fields here should be type stable.